src/pages/[platform]/build-a-backend/auth/advanced-workflows/index.mdx

```diff
@@ -396,7 +396,7 @@ import {
 } from 'aws-amplify/auth';
 
 // Note: This example requires installing `@aws-sdk/client-cognito-identity` to obtain Cognito credentials
-// npm i @aws-sdk/client-cognito-identity
+// npm add @aws-sdk/client-cognito-identity
```

Comment:
why npm add?

Reply:
it's an alias of install, but this was changed for consistency with the other pages and it translates well to other package managers
```
aliases: add, i, in, ins, inst, insta, instal, isnt, isnta, isntal, isntall
```